cmd/rook/rook.go

```diff
@@ -67,3 +57,22 @@ func addCommands() {
 func NewTableWriter(buffer io.Writer) *tabwriter.Writer {
 	return tabwriter.NewWriter(buffer, outputMinWidth, outputTabWidth, outputPadding, outputPadChar, 0)
 }
+
+func enableLogging() {
+	debugDir := os.Getenv(debugLogVar)
```

Comment:
yes, that was a side effect noticed after the change. Most commands don't have output to worry about. I actually only saw it affect one of the commands in the sample after the fact and it didn't affect the tables on the commands I ran. We will need another fix for this though.